pwa/src/templates/templateParts/dependenciesTemplates/relationsDependenciesTemplate/RelationsDependenciesTemplate.tsx

```diff
@@ -1,12 +1,13 @@
 import * as React from "react";
```

Comment:
The layout sometimes "jumps", due to the frame re-rendering itself. Even though it keeps the same nodes. A possible solution would be to check if the components have changed _and_ if there's already a frame available _before_ re-rendering anything.

This idea of something that can only exist once is called a "Singleton", this might help (no idea if it will, didn't have time to really check it out): https://www.npmjs.com/package/react-singleton-hook.